api/v1beta1/hostedcluster_types.go

```diff
@@ -20,6 +20,10 @@ const (
 	DisablePKIReconciliationAnnotation        = "hypershift.openshift.io/disable-pki-reconciliation"
 	IdentityProviderOverridesAnnotationPrefix = "idpoverrides.hypershift.openshift.io/"
 	OauthLoginURLOverrideAnnotation           = "oauth.hypershift.openshift.io/login-url-override"
+	// Priority class
+	ControlPlanePriorityClass = "hypershift.openshift.io/control-plane-priority-class"
+	APICriticalPriorityClass  = "hypershift.openshift.io/api-critical-priority-class"
```

Comment:
Let's comment each annotation with their definition in line with priority section [here](https://hypershift-docs.netlify.app/how-to/distribute-hosted-cluster-workloads/), e.g.
```
// APICriticalPriorityClass is for Pods that are required for API calls and resource admission to succeed. This includes pods like kube-apiserver, aggregated API servers...
```

Reply:
Yup, I'll get that added in. Thanks!